tehuti.py

```diff
@@ -168,6 +170,45 @@ def run(self):
         return [sum(vals) / self.number for vals in self.usage_log]
 
 
+class AccuracyMetric(Metric):
+    class Context(object):
+        pass
+
+    def __init__(self, body, setup=None, repeat=100, number=1, name=None):
```

Comment:
So I thought about this too. I decided to leave it because:

 * it follows the TimeMetric model (your previous comment notwithstanding!), and, more significantly,
 * there may be variability in the results array, which, if there is, it would be good to be able to capture. Perhaps we don't need the two repeats though...